.github/workflows/pytest-gpu.yml

```diff
@@ -59,7 +63,9 @@ jobs:
 
     - name: Test with pytest
       run: |
-        pgaccelinfo
```

Comment:
@italoaug Can you confirm that this command still does not work on the openmp runner?

Reply:
Not sure I got it. Should you be asking about `gpu-runner-04`, there is no PGI compiler installed in it.

Reply:
Yes, that's what I was asking. Thanks for clarifying.